client/graphql/fragments/userSets.graphql

```diff
@@ -0,0 +1,40 @@
+fragment userSets on User {
```

Comment:
I know we use `customSet` and `build` kind of interchangeably in the code, but could we stick to one of those and not add `sets` to the mix, i.e. rename this as `userCustomSet` or `userBuild`? Also, I think you could just have a `abbreviatedCustomSet` fragment or something like that instead of having this only be useful for a user's custom sets (so, everything inside `node` without the pageInfo).